pages/rs_quality/fair.md

```diff
@@ -1,5 +1,178 @@
 ---
-title: FAIR
+title: FAIR Research Software
 ---
 
-Tools and good enough practices for improving RS FAIRness
\ No newline at end of file
+
+## FAIR Research Software
+
+FAIR stands for Findable, Accessible, Interoperable, and Reusable and comprises a set of principles designed to
+increase the visibility and usefulness of your research to others.
+The FAIR data principles, first published [in 2016][fair-data-principles], are widely known and applied today.
+Similar [FAIR principles for software][fair-principles-research-software] have now been defined too. In general, they mean:
+
+- **Findable** - software and its associated metadata must be easy to discover by humans and machines.
+- **Accessible** - in order to reuse software, the software and its metadata must be retrievable by standard protocols, free and legally usable.
+- **Interoperable** - when interacting with other software it must be done by exchanging data and/or metadata through
+  standardised protocols and application programming interfaces (APIs).
+- **Reusable** - software should be usable (can be executed) and reusable
+  (can be understood, modified, built upon, or incorporated into other software).
+
+Each of the above principles can be achieved by a number of practices listed below.
+This is not an exact science, and by all means the list below is not exhaustive,
+but any of the practices that you employ in your research software workflow will bring you
+closer to the gold standard of fully reproducible research.
+
+### Findable
+- Create a description of your software to make it discoverable by search engines and other search tools
+- Place your software in a public software repository (and ideally register it in a [general-purpose or domain-specific software registry][software-registries])
+- Use a unique and persistent identifier (DOI) for your software (e.g. by depositing your code on [Zenodo][zenodo]), 
```

Comment:
It's not your software, but your software code release, right? (or software code). We should mention Software Heritage here (https://www.softwareheritage.org/).

Reply:
Yes, technically it is the release that gets a DOI.

Reply:
Fixed via 88db083e4868b269066ee48e625ad94e198f646c